packages/blade/src/components/FileUpload/__tests__/FileUpload.ssr.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { FileUpload } from '../FileUpload';
+import renderWithSSR from '~utils/testing/renderWithSSR.web';
+
+describe('<FileUpload />', () => {
+  it('should render in SSR', () => {
+    const { container, getByText } = renderWithSSR(
+      <FileUpload
+        uploadType="single"
```

Comment:
@anuraghazra  types are failing here, can you help?

<img width="935" alt="Screenshot 2024-02-28 at 11 17 01 AM" src="https://github.com/razorpay/blade/assets/46647141/4397cf94-4bc2-41a7-a26a-fe891a325ddb">

Reply:
Fixed